code/data_structures/src/list/singly_linked_list/operations/PalindromeInASinglyLinkedList.c

```diff
@@ -0,0 +1,62 @@
+#include<stdio.h>

```

Comment:
The filename should be all lowercase + separated by underscores.

Reply:
By filename, do you mean the "PalindromeInSinglyLinkedList.c" ?

Reply:
@iashreya Yes.